### ISSUE TITLE
Submit riot .tag syntax to linguist

### ISSUE THREAD

**Issue by GianlucaGuarini** [riot/syntax-highlight#5]
Follow up of: https://github.com/riot/riot/issues/1953

**Comment by crisward** [comment]
Working on this now. The colour used is too close to others, so I may have to change it a little.

**Comment by crisward** [comment]
I've created a pull request. Had to use the colour `#A71E49` which is not ideal, but there was an overlap with Objective-J and other 'red' languages.

**Comment by crisward** [comment]
https://github.com/github/linguist/pull/3720

**Comment by GianlucaGuarini** [comment]
@crisward thanks for your pull request but regarding the color I would like to keep `#fc0e49` unless linguist requires a different color by specs. Please fix also the [`missing license data`](https://travis-ci.org/github/linguist/jobs/254209716) issue as well. Good job

**Comment by crisward** [comment]
Yes, I'm sorry to say the spec prevent choosing that color. I tried changing it a little at a time, but couldn't get anything to pass the tests that are even close to riot's color.

I'll take a look at the licence data.

**Comment by GianlucaGuarini** [comment]
@crisward I couldn't find anything about colors specs [here](https://github.com/github/linguist/blob/master/CONTRIBUTING.md)

**Comment by crisward** [comment]
The colors are set here - https://github.com/crisward/linguist/blob/master/lib/linguist/languages.yml#L3759

When the test are run it checks how distinct they are. If they're not different enough from existing colors the tests fail and can't be merged.

I've added another LICENCE file to the language repository, but not sure how to get the tests to run again without making a random commit.

**Comment by GianlucaGuarini** [comment]
your pull request should be automatically updated when you push against master.. regarding the color decision maybe it will be worth pinging @JLehtinen
